app/monitoring/StateMachineMonitor.scala

```diff
@@ -0,0 +1,21 @@
+package monitoring
+
+import akka.actor.ActorSystem
+import services.stepfunctions.RegularContributionsClient
+import scala.concurrent.duration._
+import scala.util.{Failure, Success}
+import SafeLogger._
+
+class StateMachineMonitor(client: RegularContributionsClient, actorSystem: ActorSystem) {
+
+  def start(): Unit = {
+    implicit val ec = actorSystem.dispatcher
+    actorSystem.scheduler.schedule(5.seconds, 60.seconds) {
+      client.healthy.onComplete {
+        case Success(true) => SafeLogger.debug("Regular contributions state machine is healthy")
+        case Success(false) => SafeLogger.error(scrub"Regular contributions state machine is not available")
```

Comment:
I'm slightly worried about someone re-phrasing this and breaking the monitoring you've implemented here. 

Some things we could do to mitigate against this:

1. Add a comment to note that we have a metric with a dependency on this precise String. 
2. Make it a val and imply the above (in the name).
3. Make it look a bit more machine-y 'regular-contributions-step-function-unavailable' to help people fill in the gaps.